AgentInterfaces/AgentInterfaces.csproj

```diff
@@ -9,7 +9,7 @@
     -->
     <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
     <PackageId>PlayFab.MultiplayerServers.AgentInterfaces</PackageId>
-    <PackageVersion>5.1.5</PackageVersion>
+    <PackageVersion>5.2.0</PackageVersion>
```

Comment:
why not 5.1.6? :)

Reply:
This change is the definition of a minor version change--it's a backwards-compatible change to the public API (i.e. i added a field to SessionHostHeartbeatInfo)